tutorials/fundamentals/unshift.md

```diff
@@ -0,0 +1,7 @@
+The `unshift()` function adds one or more elements to the beginning of the array and returns the new length of the array.
+
+```javascript
+const array = [1,2,3,4,5];
+array.unshift(-5,-4,-3,-2,-1,0); // 11
```

Comment:
Make `const array = [3, 4, 5]` and `unshift(1, 2)`. `1, 2, 3, 4, 5` is easier to read than `-5 ...`